src/services/transactions/trackTransactions.ts

```diff
@@ -0,0 +1,23 @@
+import { SendTransactionReturnType, TrackTransactionsPropsType } from 'types';
+import { trackServerTransactions } from './trackServerTransactions';
+
+export async function trackTransactions({
```

Comment:
Why not simply not use this helper and keep only the one above? This seems to be a wrapper around it